index.ts

```diff
@@ -163,15 +196,19 @@ wss.on("request", function (request) {
 
     var data = JSON.parse(message.utf8Data);
     // console.log(`Inbound: ${message.utf8Data}`);
-    let card = undefined;
+    let card: Card = undefined;
 
     switch (data.type) {
       case "join":
         do {
           id = generateId();
         } while (room.connections.filter((item) => item.id == id).length > 0);
 
-        room.connections.push({ name: data.name, connection, id });
+        room.connections.push({
+          name: data.name,
+          connection: connection,
+          id: id,
+        });
```

Comment:
What is this change required for? The line only gets longer and is the reformatted. Has it to do with the `typescript` switch?

Reply:
husky as well. I did no such expansions.